pages/database-management/monitoring.mdx

```diff
@@ -137,20 +263,30 @@ All system metrics measuring different parts of the system can be divided into t
  | average_degree | Gauge (double)   | Average number of relationships of a single node.           |
  | memory_usage   | Gauge (uint64_t) | Amount of RAM used reported by the OS (in bytes).           |
  | disk_usage     | Gauge (uint64_t) | Amount of disk space used by the [data directory](/fundamentals/data-durability) (in bytes). |
+ | peak_memory_usage | Gauge (uint64_t) | Peak RAM memory usage in the system during the whole run (in bytes).  |
 
 #### Index metrics
 
  | Name                       | Type    | Description                                            |
  | -------------------------- | ------- | ------------------------------------------------------ |
  | ActiveLabelIndices         | Counter | Number of active label indexes in the system.          |
  | ActiveLabelPropertyIndices | Counter | Number of active label property indexes in the system. |
+ | ActiveTextIndices          | Counter | Number of active text indexes in the system.           |
```

Comment:
I know I am late to the party, but does anyone know why are these `Counter`? This should be `Gauge` because it can go up or down. Counter should always go up or have a reset to 0.

> A counter is a cumulative metric that represents a single [monotonically increasing counter](https://en.wikipedia.org/wiki/Monotonic_function) whose value can only increase or be reset to zero on restart.